src/protagonist/Engine/Ingest/Timebased/TranscodeCompleteHandler.cs

```diff
@@ -13,7 +14,10 @@ public class TranscodeCompleteHandler : IMessageHandler
 {
     private readonly ITimebasedIngestorCompletion timebasedIngestorCompletion;
     private readonly ILogger<TranscodeCompleteHandler> logger;
-    private static readonly JsonSerializerOptions Settings = new(JsonSerializerDefaults.Web);
+    private static readonly JsonSerializerOptions Settings = new(JsonSerializerDefaults.Web)
+    {
+       NumberHandling = JsonNumberHandling.WriteAsString

```

Comment:
Should this be `NumberHandling = JsonNumberHandling.AllowReadingFromString` and the attribute can be removed from the model?